Gemfile.lock

```diff
@@ -261,6 +271,7 @@ GEM
     rb-fsevent (0.10.3)
     rb-inotify (0.10.0)
       ffi (~> 1.0)
+    rb-readline (0.5.5)
```

Comment:
Did this get in here from `prawn` or do we need to rerun `bundle` to get a fresh `Gemfile.lock`?

Reply:
I can remove it if you prefer we do. I can also just add it in development mode to spare some people the trouble?